Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_2_5.md

```diff
@@ -0,0 +1,6 @@
+ 
+#### Integrations
+
+##### EWS v2
+
+Added support for the *handle_inline_image* argument in the ***reply-mail*** command. Whether to handle inline images in the HTML body.
```

Comment:
Added support for the *handle_inline_image* argument in the ***reply-mail*** command. This enables you to include inline images in the HTML body of the email.